lib/server.js

```diff
@@ -102,9 +102,9 @@ function parsePOST(req) {
 
 function logResponse(opts, response, startTime) {
     const latency = Date.now() - startTime;
-    let logLevel = 'trace/request';
+    let logLevel = 'info/request';
```

Comment:
We could consider slimming down on the response headers, similar to how we whitelist request headers. Either way though, this is always going to be structured data & typically consumed / analyzed using tools like logstash, jq, or [the bunyan cli](https://github.com/trentm/node-bunyan#cli-usage).